ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/text/v1/text/clientlibs/site/js/text.js

```diff
@@ -0,0 +1,113 @@
+/*******************************************************************************
+ * Copyright 2022 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+(function() {
+    "use strict";
+
+    var dataLayerEnabled;
+    var dataLayer;
+
+    /**
+     * Adds Click Event Listener to the main <div> of the Text Components
+     */
+    function addClickEventListenerToTextComponents() {
+        var componentMainDivs = document.getElementsByClassName("cmp-text");
```

Comment:
are we adding support for data layer 

@nit23uec ?

Reply:
No, we should remove this.

Reply:
removed.